src/schema/v2/marketingCollections.ts

```diff
@@ -320,6 +324,24 @@ export const MarketingCollections: GraphQLFieldConfig<void, ResolverContext> = {
   }),
   resolve: async (_root, args, { marketingCollectionsLoader }) => {
     const { size } = convertConnectionArgsToGravityArgs(args)
+    // Enable curated sorting
+    if (args.sort === MARKETING_COLLECTIONS_SORTS.CURATED.value) {
```

Comment:
**suggestion / non-blocking**: Just realized that this would accomplish the same thing through the already exported enum type:

```ts
if (args.sort === MarketingCollectionsSorts.getValue('CURATED')?.value) {
```